drivers/SPI.h

```diff
@@ -77,22 +82,22 @@ class SPI : private NonCopyable<SPI> {
 
 public:
 
-    /** Create a SPI master connected to the specified pins
+    /** Create a SPI master connected to the specified pins.
      *
-     *  mosi or miso can be specified as NC if not used
+     *  @note Either mosi or miso can be specified as NC if not used.
      *
-     *  @param mosi SPI Master Out, Slave In pin
-     *  @param miso SPI Master In, Slave Out pin
-     *  @param sclk SPI Clock pin
-     *  @param ssel SPI chip select pin
+     *  @param mosi SPI Master Out, Slave In pin.
+     *  @param miso SPI Master In, Slave Out pin.
+     *  @param sclk SPI Clock pin.
+     *  @param ssel SPI Chip Select pin.
      */
     SPI(PinName mosi, PinName miso, PinName sclk, PinName ssel = NC);
     virtual ~SPI();
 
-    /** Configure the data transmission format
+    /** Configure the data transmission format.
```

Comment:
Do these need \brief commands/tags/whatever they're called?

Reply:
No. @brief is implied.

Reply:
👍